snap/local/files/run_dcgm_exporter.sh

```diff
@@ -4,11 +4,17 @@ set -euo pipefail
 # Build the argument list for the dcgm-exporter command
 args=()
 
+# Check if the nv-hostengine-port option is set.
```

Comment:
It's always setting, isn't? I think we can drop this comment and also the one about  `dcgm-exporter-address`